htdocs/api/v0.0.1/Login.php

```diff
@@ -129,6 +133,13 @@ function getEncodedToken($user)
                  );
 
         $key = $config->getSetting("JWTKey");
+        if (!isKeyStrong($key)) {
+            error_log(
+                'ERROR: JWTKey config variable is weak. '
+                .'Please change the key to a more cryptographically-secure value.'
+            );
+            return "";
```

Comment:
HTTP 500 would be better beacause returning an empty string wont allow to go further. And it more like a server config error IMHO